Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/ContextMtl.mm

```diff
@@ -1648,6 +1653,33 @@ GLint GetOwnershipIdentity(const egl::AttributeMap &attribs)
     return angle::Result::Stop;
 }
 
+angle::Result ContextMtl::bindMetalRasterizationRateMap(gl::Context *context,
+                                                        RenderbufferImpl *renderbuffer,
+                                                        GLMTLRasterizationRateMapANGLE map)
+{
+    id<MTLRasterizationRateMap> rateMap = (__bridge id<MTLRasterizationRateMap>)(map);
+    if (rateMap && rateMap.device != mContextDevice.get())
+    {
```

Comment:
IDK about angle's style rules but you've got some inconsistency here with the braces

Reply:
oh thank you, they should all be on separate lines